client/src/components/App6Symbol.tsx

```diff
@@ -0,0 +1,145 @@
+import { App6Choices } from "components/App6"
+import ms from "milsymbol"
+import React, { useEffect, useRef } from "react"
+
+const VERSION = 10
+
+export const getChoices = (field: string, values: any[]) => {
+  const symbolSet = getCodeFieldValue(getSymbolCode(values), "app6symbolSet")
+  const app6entity = getCodeFieldValue(getSymbolCode(values), "app6entity")
+  const app6entityType = getCodeFieldValue(
+    getSymbolCode(values),
+    "app6entityType"
+  )
+
+  const toEntries = options =>
+    Object.fromEntries(
+      Object.entries(options).map(([key, value]) => [key, value.label])
+    )
+
+  const choiceHandlers = {
+    app6entity: () => toEntries(App6Choices[field][symbolSet] || {}),
+    app6entityType: () =>
+      toEntries(App6Choices.app6entity[symbolSet]?.[app6entity]?.options || {}),
+    app6entitySubtype: () => {
+      return toEntries(
+        App6Choices.app6entity[symbolSet]?.[app6entity]?.options?.[
+          app6entityType
+        ]?.options || {}
+      )
+    },
+    app6amplifier: () => App6Choices[field][symbolSet] || {},
+    app6sectorOneModifier: () => App6Choices[field][symbolSet] || {},
+    app6sectorTwoModifier: () => App6Choices[field][symbolSet] || {}
+  }
+
+  return (choiceHandlers[field] || (() => App6Choices[field] || {}))()
+}
+
+export const getLabel = (values: any[], field: string) => {
+  const choices = getChoices(field, values)
+  const value = values[field]
+  return choices[value] || ""
+}
+
+export const getFieldsList = () => {
+  return [
+    "app6context",
+    "app6standardIdentity",
+    "app6symbolSet",
+    "app6hq",
+    "app6amplifier",
+    "app6entity",
+    "app6entityType",
+    "app6entitySubtype",
+    "app6sectorOneModifier",
+    "app6sectorTwoModifier"
+  ]
+}
```

Comment:
This is simply a constant array, no need to call a function for it, so something like:
```javascript
export const fieldsList = [
  "app6context",
  "app6standardIdentity",
  "app6symbolSet",
  "app6hq",
  "app6amplifier",
  "app6entity",
  "app6entityType",
  "app6entitySubtype",
  "app6sectorOneModifier",
  "app6sectorTwoModifier"
]
```
will do fine.